helpers/foundation-deployer/stages/destroy.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stages
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/mitchellh/go-testing-interface"
+
+	"github.com/terraform-google-modules/terraform-example-foundation/helpers/foundation-deployer/steps"
+	"github.com/terraform-google-modules/terraform-example-foundation/helpers/foundation-deployer/utils"
+)
+
+const (
+	MaxBuildRetries = 40
+)
+
+func DestroyBootstrapStage(t testing.TB, s steps.Steps, c CommonConf) error {
+	repo := "gcp-bootstrap"
+	step := "0-bootstrap"
```

Comment:
Should we move these to consts since we use the in `DeployBootstrapStage` too?

Reply:
done